src/model/cloud-runner/services/cloud-runner-system.ts

```diff
@@ -0,0 +1,44 @@
+import { exec } from 'child_process';
+import { RemoteClientLogger } from '../remote-client/remote-client-logger';
+export class CloudRunnerSystem {
```

Comment:
Please use an empty line after imports.

I'll introduce an ESLint rule that enforces this if it isn't already.